test/data/test_dataset.py

```diff
@@ -85,7 +86,10 @@ def test_dataset_inductive_transform():
 
 
 def test_converter():
-    dataset = FakeDataset(num_rows=10).materialize()
+    dataset = FakeDataset(
+        num_rows=10,
+        stypes=[stype.categorical, stype.numerical,
+                stype.multi_categorical]).materialize()
```

Comment:
can we name this `multicategorical`?

Reply:
🤣 Just discussed with @weihua916 yesterday and he thinks `multi_categorical` is better.